### PR TITLE
Add Atum, Nature's Aura and Ice & Fire support

### PR DESCRIPTION
**Pull request by Azmalent**
I added support for three mods:
- Atum 2
    - Palm planks
    - Deadwood planks
    - Limestone
    - Alabaster
    - Porphyry 
- Nature's Aura
    - Ancient planks
    - Infused rock
- Ice & Fire:
    - Charred cobblestone
    - Charred stone

### CODE REVIEW

**Comment by asiekierka** [comment]
Awesome!

File: src/main/resources/assets/unlimitedchiselworks/ucwdefs/chisel/iceandfire.json [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+          "variation"
+        ]
+      },
+      "based_upon": {
+        "state": "minecraft:cobblestone"
+      }
+    },
+	{
+      "from": {
+        "block": "iceandfire:chared_stone"
```
Comment (asiekierka) [inline src/main/resources/assets/unlimitedchiselworks/ucwdefs/chisel/iceandfire.json]: Is that a stone brick, or just a stone? Gotta be a stone brick.

**Comment by asiekierka** [comment]
Ice and Fire support looks terrible, so I culled it for now; the rest has been merged.

File: src/main/resources/assets/unlimitedchiselworks/ucwdefs/chisel/iceandfire.json (hunk 1 above)
Comment (Azmalent) [reply src/main/resources/assets/unlimitedchiselworks/ucwdefs/chisel/iceandfire.json]: It's just plain stone, there are no bricks. It seemed to work okay, though.

**Pull request closed without merging**